data/pug.txt

```diff
@@ -317,6 +317,16 @@ outfit "Pug Biodefenses"
 	"unplunderable" 1
 	description "This defense system is designed to release a suite of toxins that are lethal to human beings (and probably to some other species, as well) but which have no effect at all on the Pug."
 
+outfit "Pug Peacekeeping Staff"
+	plural "Pug Peacekeeping Staves"
+	category "Hand to Hand"
+	cost 11000
```

Comment:
Laser Rifle: 8k @ .6 attack
Pulse Rifle: 11k @ .7 attack
Tuning Rifle: 65k @ 1.2 attack
Frags: 17k @ 1.3 attack
Korath Rifle: 73k @ 1.6 attack

1) This should probably be more expensive
2) I think we should have a look at the cost of some of the new H2H vs its attack.

Reply:
I'd say 102k for cost.